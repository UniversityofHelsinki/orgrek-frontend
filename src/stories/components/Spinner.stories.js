import SpinnerComponent from '../../components/Spinner';

export default {
  component: SpinnerComponent,
};

export const Spinner = {};
