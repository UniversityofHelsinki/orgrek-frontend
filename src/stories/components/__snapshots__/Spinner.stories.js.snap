// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`stories/components/Spinner Spinner smoke-test 1`] = `
Object {
  "checked": undefined,
  "children": undefined,
  "name": "stories-components-spinner--spinner",
  "pressed": undefined,
  "role": "WebArea",
  "value": undefined,
}
`;
